### DISCUSSION TITLE
How to save checkpoint to a directory which is not in current working directory?

### DISCUSSION THREAD

**Discussion by ZeguanXiao** [Lightning-AI/pytorch-lightning#11293] [Lightning Trainer API: Trainer, LightningModule, LightningDataModule]
I have limited space on the disk which my code is stored. Now I use ModelCheckpoint Callback to automatically save my checkpoints. I read the [document](https://pytorch-lightning.readthedocs.io/en/latest/extensions/generated/pytorch_lightning.callbacks.ModelCheckpoint.html) about this callback and it seems I can only save checkpoint in the current working directory?

**Comment by rohitgr7** [comment]
you can specify `dirpath` to save it anywhere, even on a remote disk-like s3 or gcloud.
